pkgs/development/python-modules/ircrobots/default.nix

```diff
@@ -13,21 +13,19 @@
 
 buildPythonPackage rec {
   pname = "ircrobots";
-  version = "0.4.6";
+  version = "0.6.5";
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "jesopo";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-+BrS1+ZkgwT/qvqD0PwRZi2LF+31biS738SzKH1dy7w=";
+    sha256 = "111383anz834kc9mhmi0yvsvc95qfj0xw17v95vp1pqn973v6n56";
```

Comment:
```
❯ nix hash to-sri --type sha256 111383anz834kc9mhmi0yvsvc95qfj0xw17v95vp1pqn973v6n56
sha256-plizx0kW33B3SfsE3oF0uCS29fYgVlgTm2Sgb9VAI4Q=
```

Let's make sure we use `hash` everywhere. We actively migrate towards it in python-modules.

Reply:
:+1:, is there an easy way for me to have Nix 2.3 output SRI hashes by default to avoid this happening in the future? I don't mind running a patched version.

Reply:
I don't know.

Reply:
Fixed (and changed the existing instance of `sha256` to `hash` while I was at it).